Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -44,36 +67,65 @@ ES_PORT='9200'
 
 CURL_N_MAX=10
 SLEEP=5
-DELIMITER=`echo -e -n "\x00"`
-EOProcess=`echo -e -n "\x06"`
+EOBatch='\x17'
+BATCHMODE="d"
+
+case $BATCHMODE in
+  e|enabled)
+    [ -z "$EOB" ] && EOBatch="\n"
+    ;;
+  d|disabled)
+    ( [ -z "$EOB" ] || [ $EOB == "\x17" ] || [ $EOBatch == "\x17" ] ) && EOBatch="\n"
+    ( [ -n "$EOB" ] && [ ! $EOB == "\x17" ] ) && EOBatch="$EOB"
+    ;;
+  *)
+    log ERROR "Unexpected batch-mode parameter." && { [ -n "$BATCHMODE" ] && usage && return 2; }
+    break
+    ;;
+esac
 
 cmd="curl $ES_AUTH http://$ES_HOST:$ES_PORT/_bulk?pretty --data-binary @"
 
 load_files () {
-  [ -z "$1" -o ! -f "$1" ] && log $(usage) && exit 1
+  [ -z "$1" -o ! -f "$1" ] && log NOTSET $(usage) && exit 1
```

Comment:
Why `NOTSET`? I\`d say `INFO` or `ERROR` or, if the idea is to output `usage` without log level -- use `usage >&2`, plain `usage`, or maybe even teach `log` to work with skipped level parameter...

Reply:
@mgolosova 

I couldn't decide which level (`ERROR` or `INFO`) that case belongs to. I decided to assign to it level 0 (NOTSET). I took into account PR #135, so why not to mention NOTSET directly and allow to handle log with skipped parameter as level 0?